doc/authors.rst

```diff
@@ -1,22 +1,31 @@
-========================
+.. _authors:
+
+************************
 Authors and contributors
-========================
+************************
 
 The following people have contributed code and/or ideas to the current version
-of Elephant. The institutional affiliations are those at the time of the
+of elephant. The institutional affiliations are those at the time of the
```

Comment:
I thought we discussed this a long time ago, and decided it should be "Elephant", not "elephant" for the project name (although of course, we still `import elephant`).